examples/async-universal/server/server.js

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable no-console, no-use-before-define */
```

Comment:
It is confusing that this single example uses CommonJS in server, and all other examples use ES6. This inconsistency is surprising. I understand why you did this but I’d say we should write this file in CommonJS for consistency’s sake. Later we can change all examples to use ES6 on the server as a single chunk of work.